CNAME

```diff
@@ -1 +0,0 @@
-adaway.org
```

Comment:
Isn't this configuration needed?

Reply:
Yep it's needed if you still want to use the domain. btw: @PerfectSlayer if you like to take over the domain, email me.